docs/configuration.md

```diff
@@ -243,6 +243,47 @@ values, but here they are if you need to:
 ### regex
 This is the regex which is used to match the current branch to the correct branch configuration.
 
+### source-branches
+Because git is a directed graph GitVersion sometimes cannot tell which branch the current branch was branched from.
```

Comment:
> Because git is a directed graph...

I'm guessing that a significant number of people won't know what this means, and it seems redundant. Perhaps remove it?